use-timescale/query-data/index.md

```diff
@@ -9,7 +9,7 @@ keywords: [queries]
 
 Timescale hypertables _are_ PostgreSQL tables. That means you can query them
 with standard SQL commands. In addition to the extra benefits you get from Timescale architecture and query planning,
-use [PopSQL][popsql] to work on data with centralized SQL queries, interactive visuals and real-time collaboration
+use [PopSQL](https://popsql.com/) to work on data with centralized SQL queries, interactive visuals and real-time collaboration
```

Comment:
It will be more helpful to link to https://docs.timescale.com/getting-started/latest/run-queries-from-console/, so could you please:
- revert the change to the link structure, so [PopSQL][popsql] (this is a reference-style link and this is proper formatting)
- add the following at the very bottom of the page, under `[skipscan]`: 
    [popsql]: /getting-started/:currentVersion:/run-queries-from-console/

Thanks!